Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml

```diff
@@ -0,0 +1,139 @@
+- type: entity
+  name: BaseWeaponRailgun
+  parent: BaseItem
+  id: BaseWeaponRailgun
+  description: A terrifying thunderous railcannon made handheld.
+  abstract: true
+  components:
+  - type: Item
+    size: Huge
+  - type: Clothing
+    sprite: DeltaV/Objects/Weapons/Guns/Railguns/railgun.rsi
+    quickEquip: false
+    slots:
+    - Back
+  - type: AmmoCounter
+  - type: Wieldable
+    wieldSound:
+      path: /Audio/DeltaV/Effects/railgun_charge.ogg
+  - type: UseDelay
+    delay: 5
+  - type: HeldSpeedModifier
+    walkModifier: 0.75
+    sprintModifier: 0.75
+  - type: GunRequiresWield
+  - type: Gun
+    fireRate: 0.20
+    selectedMode: SemiAuto
+    availableModes:
+    - SemiAuto
+    soundGunshot:
+      path: /Audio/DeltaV/Weapons/Guns/Gunshots/railgun_laser.ogg
+  - type: BallisticAmmoProvider
+    cycleable: false
+    capacity: 3
+    proto: BulletRailgunBasic
+    whitelist:
+      tags:
+        - BulletRailgun
+  - type: ContainerContainer
+    containers:
+      ballistic-ammo: !type:Container
+        ents: []
+  - type: StaticPrice
+    price: 10000
+
+- type: entity
+  name: Decimator Railgun
+  parent: BaseWeaponRailgun
+  id: WeaponRailgunDecimator
+  description: A terrifying starship railcannon, now in your hands.
+  components:
+  - type: BallisticAmmoProvider
+    cycleable: false
+    capacity: 3
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Guns/Railguns/railgun.rsi
+    layers:
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+  - type: Wieldable
+  - type: GunRequiresWield
+    wieldSound:
+      path: /Audio/DeltaV/Effects/railgun_charge.ogg
```

Comment:
this should come under - type: Wieldable

this is also defined in the parent: BaseWeaponRailgun, so is a duplicate anyways